src/api_server/swagger/firecracker.yaml

```diff
@@ -531,19 +531,41 @@ definitions:
       - C3
       - T2
 
+  CreateSnapshotParams:
+    type: object
+    required:
+      - mem_file_path
+      - snapshot_path
+    properties:
+      mem_file_path:
+        type: string
+        description: Path to the file that will contain the guest memory.
+      snapshot_path:
+        type: string
+        description: Path to the file that will contain the microVM state.
+      snapshot_type: string
```

Comment:
```suggestion
      snapshot_type:
        type: string
      
```

Reply:
As above: I'd prefer to keep this PR to simple reordering.

Reply:
I've opened issue #1905 to address these issues.

Reply:
Thanks for opening the issue! It's true that we should have a mechanism for automatically detecting problems in the API file. Though unrelated to your PR, I requested the changes because they are small and your code was very close to them :)